main.go

```diff
@@ -35,7 +35,9 @@ import (
 
 	"github.com/ramendr/ramen/controllers"
 	rmnutil "github.com/ramendr/ramen/controllers/util"
+
 	// +kubebuilder:scaffold:imports
+	Recipe "github.com/ramendr/recipe/api/v1alpha1"
```

Comment:
I'm not familiar with the `[kubebuilder:scaffold](https://github.com/kubernetes-sigs/kubebuilder/issues/1487)` marker.  Any reason why the recipe import is after it?

Reply:
Copy/paste issue. Removing this didn't seem to have any negative effect.